spec/controllers/event_steps_controller_spec.rb

```diff
@@ -1,8 +1,18 @@
 require "rails_helper"
```

Comment:
I don't feel strongly about it, but I would expect request specs to live in `/spec/requests/`

Reply:
Yeh, good call - I'll move all of them.

Reply:
Ok moved.